### PR TITLE
feat: add kclvm-windows.zip build to windows github action

### PR DESCRIPTION
**Pull request by chai2010**
build kclvm-windows.zip in github action

### CODE REVIEW

File: kclvm/runner/src/command.rs [hunk 1]
```diff
@@ -381,12 +381,22 @@ impl Command {
     }
 
     fn get_rust_libstd_dylib(executable_root: &str) -> String {
-        let txt_path = std::path::Path::new(&executable_root)
-            .join("lib")
-            .join("rust-libstd-name.txt");
-        let rust_libstd_name = std::fs::read_to_string(txt_path).unwrap();
-        let rust_libstd_name = rust_libstd_name.trim();
-        format!("{}/lib/{}", executable_root, rust_libstd_name)
+        if Self::is_windows() {
```
Comment (Peefy) [inline kclvm/runner/src/command.rs:384]: let txt_path = std::path::Path::new(&executable_root)
                .join(if Self::is_windows() {"libs"} else {"lib"})
                .join("rust-libstd-name.txt");
            let rust_libstd_name = std::fs::read_to_string(txt_path).unwrap();
            let rust_libstd_name = rust_libstd_name.trim();
            format!("{}/libs/{}", executable_root, rust_libstd_name)

Comment (chai2010) [reply kclvm/runner/src/command.rs]: fixed

**Review by Peefy — APPROVED** [review]
LGTM

**Review by elliotxx — APPROVED** [review]
lgtm

**Pull request merged**